app/Options.hs

```diff
@@ -31,7 +32,7 @@ data LifeCommand
 commandParser :: Parser LifeCommand
 commandParser = subparser $
     command "init"
-            (info (helper <*> fmap Init initOptionsParser)
+            (info (helper <*> fmap Init ownerParser)
```

Comment:
That's nice decision as we won't need anything else but `username` for initialisation